pytest_mh/utils/hostname.py

```diff
@@ -0,0 +1,101 @@
+"""Manage system hostname using the 'hostname' command."""
+
+from __future__ import annotations
+
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.conn import ProcessResult
+
+__all__ = ["HostnameUtils"]
+
+
+class HostnameUtils(MultihostUtility[MultihostHost]):
+    """
+    Provides a Python wrapper for the basic 'hostname' command-line utility.
+    """
+
+    def __init__(self, host: MultihostHost) -> None:
+        """
+        Initialize the HostnameUtils.
+
+        :param host: The multihost host instance.
+        :type host: MultihostHost
+        """
+        super().__init__(host)
+
+    @property
+    def name(self) -> str:
```

Comment:
I think this should be part of the client role. So it's easier to access when writing tests. 

```client.hostname```

Reply:
Check previous reply